### PR TITLE
Feat/optional number row

### PR DESCRIPTION
**Pull request by ismailnurudeen**

### CODE REVIEW

File: app/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <declare-styleable name="MyKeyboard">
         <attr name="keyWidth" format="fraction" />
         <attr name="horizontalGap" format="fraction" />
+        <attr name="isNumRow" format="boolean" />
```
Comment (tibbi) [inline app/src/main/res/values/attrs.xml:14]: change it to "isNumbersRow" please

Comment (ismailnurudeen) [reply app/src/main/res/values/attrs.xml:14]: Sure. will do that.

**Comment by tibbi** [comment]
seems to work well, but when the numbers row is enabled, can we maybe no longer show numbers at long pressing the top line q, w, e, r....? So that instead of "ř4ŕ" we would just show "řŕ"? If it needs some hacks or causes new issues, lets ignore it.

**Comment by ismailnurudeen** [comment]
Don't merge yet.

I just found a bug where the last row disappears if numbers row is enabled and you click on the `123` button and then click on the `abc` button.

File: app/src/main/kotlin/com/simplemobiletools/keyboard/helpers/MyKeyboard.kt [hunk 2]
```diff
@@ -342,14 +345,31 @@ class MyKeyboard {
...
+                            if (currentRow?.isNumbersRow == true && !context.config.showNumbersRow) {
+                                continue
+                            }
                             inKey = true
                             key = createKeyFromXml(res, currentRow!!, x, y, parser)
+                            if(context.config.showNumbersRow){
+                                // Removes numbers (i.e 0-9) from the popupCharacters if numbers row is enabled
+                                key.apply{
+                                    popupCharacters = popupCharacters?.replace(Regex("\\d+"), "")
+                                    if(popupCharacters.isNullOrEmpty()){
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/keyboard/helpers/MyKeyboard.kt:367]: spaces.. Format the code

**Comment by tibbi** [comment]
should be good, thanks

**Pull request merged**